pkg/flowaggregator/flowaggregator.go

```diff
@@ -325,64 +325,84 @@ func (fa *flowAggregator) initExportingProcess() error {
 }
 
 func (fa *flowAggregator) Run(stopCh <-chan struct{}) {
-	exportTicker := time.NewTicker(fa.exportInterval)
-	defer exportTicker.Stop()
 	go fa.collectingProcess.Start()
 	defer fa.collectingProcess.Stop()
 	go fa.aggregationProcess.Start()
 	defer fa.aggregationProcess.Stop()
+	go fa.flowRecordExpiryCheck(stopCh)
+
+	<-stopCh
+}
+
+func (fa *flowAggregator) flowRecordExpiryCheck(stopCh <-chan struct{}) {
+	expireTicker := time.NewTicker(fa.activeFlowRecordTimeout)
```

Comment:
I don't think a ticker is the best construct anymore? We essentially call `fa.aggregationProcess.GetExpiryFromExpirePriorityQueue` and set the ticker to the returned value, but we will only use the ticker for a single tick...  Maybe use a timer instead?

Reply:
Yes, the timer makes more sense and it's simpler too. Modified.